code/game/machinery/vendors/vending.dm

```diff
@@ -1118,6 +1118,14 @@
 	tilt(target, from_combat = TRUE)
 	return TRUE
 
+/obj/machinery/economy/vending/carbon_throw_hit(mob/living/carbon/human/C, datum/thrownthing/throwingdatum, damage, mob_hurt, self_hurt)
+	if(HAS_TRAIT(C, TRAIT_FLATTENED))
+		return
```

Comment:
do you just want to return here and null out any effects or just prevent tilt'ing?

Reply:
Probably should only block tilting

Reply:
can just do `return ..()` then

Reply:
done